tools/code_format/check_format.py

```diff
@@ -758,6 +758,11 @@ def checkSourceLine(self, line, file_path, reportError):
       reportError("Don't use std::variant; use absl::variant instead")
     if self.tokenInLine("std::visit", line):
       reportError("Don't use std::visit; use absl::visit instead")
+    if " try {" in line and file_path.startswith("./source") and not file_path.startswith(
+        "./source/extensions") and not file_path == "./source/common/common/thread.h":
```

Comment:
can you add a TODO to remove the source/extensions carve-out?

Reply:
Thanks, will do.